server/middlewares/proxy.ts

```diff
@@ -0,0 +1,19 @@
+const envs = require("env-smart")
+// read environment variable
+const OS_HTTP_PROXY = envs.load().HTTP_PROXY
+const OS_HTTPS_PROXY = envs.load().HTTPS_PROXY
+const proxy = () => {
+  let proxyAddress = null
+  /* istanbul ignore else */
+  if (typeof OS_HTTPS_PROXY !== "undefined") {
+    console.log("OS HTTPS_PROXY Environment variable :", `${OS_HTTPS_PROXY}`)
```

Comment:
Use `logger.debug` instead of `console.log`

Reply:
we have used logger.debug instead of console.log as you suggested.